packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss

```diff
@@ -50,34 +56,59 @@
   --cbp-tooltip-color-bg: var(--cbp-tooltip-color-bg-dark);
   --cbp-tooltip-color: var(--cbp-tooltip-color-dark);
  
-  --cbp-tooltip-color-hover: var(--cbp-tooltip-color-hover-dark);
-  --cbp-tooltip-color-bg-hover: var(--cbp-tooltip-color-bg-hover-dark);
+  --cbp-tooltip-control-color-hover: var(--cbp-tooltip-control-color-hover-dark);
+  --cbp-tooltip-control-color-bg-hover: var(--cbp-tooltip-control-color-bg-hover-dark);
+
+  --cbp-tooltip-control-color-focus: var(--cbp-tooltip-control-color-focus-dark);
+  --cbp-tooltip-control-color-bg-focus: var(--cbp-tooltip-control-color-bg-focus-dark);
 
-  --cbp-tooltip-color-focus: var(--cbp-tooltip-color-focus-dark);
-  --cbp-tooltip-color-bg-focus: var(--cbp-tooltip-color-bg-focus-dark);
+  --cbp-tooltip-definition-color: var(--cbp-tooltip-definition-color-dark);
+  --cbp-tooltip-definition-color-hover: var(--cbp-tooltip-definition-color-hover-dark);
+  
+  --cbp-tooltip-definition-color-focus: var(--cbp-tooltip-definition-color-focus-dark);
+  --cbp-tooltip-definition-outline-focus: var(--cbp-tooltip-definition-outline-focus-dark);
 }
 
 cbp-tooltip {
   display: inline-block;
   position: relative;
   --cbp-tooltip-placement-gap: var(--cbp-tooltip-gap);
   --cbp-tooltip-placement-offset: var(--cbp-tooltip-offset);
-  --cbp-link-decoration: var(--cbp-tooltip-link-decoration);
 
   &[open] [role='tooltip']{
       display: flex;
   }
     
+  &:not([definitionlinkstyle="true"]):hover{
```

Comment:
don't make this a `:not()`. Just assign the default with no attribute selector and override with the attribute selector.

Reply:
removed the :not()